SU2_SOL/src/SU2_SOL.cpp

```diff
@@ -49,6 +49,7 @@ int main(int argc, char *argv[]) {
   int size = SINGLE_NODE;
   bool fem_solver = false;
   bool periodic = false;
+  bool multizone = false;
```

Comment:
DEF and DOT have not been included in this PR. It may happen that they need some treatment, but they are not as mature for multi-zone problems as CFD and SOL. So the way we want to generalize them for MZ I think it's a matter of a future PR, as this is a multistage process and we still have some way to go.